src/phovea.ts

```diff
@@ -0,0 +1,11 @@
+import {IRegistry, asResource} from 'phovea_core/src/plugin';
+
+//register all extensions in the registry following the given pattern
+export default function (registry: IRegistry) {
+  //registry.push('extension-type', 'extension-id', function() { return System.import('./src/extension_impl'); }, {});
+  registry.push('epPhoveaCoreLocale', 'phoveaCoreLocaleEN', function () {
```

Comment:
```suggestion
  registry.push('epPhoveaCoreLocale', 'phoveaCoreSecurityFlaskLocaleEN', function () {
```

Shouldn't the extension id should be unique? Because my assumption is that the translations will be deep merged even if the id is unique.

Reply:
Independent of yes or no, we should add this point to the wiki.